snippets/perl/post_nb_cleanup_asset.pl

```diff
@@ -0,0 +1,75 @@
+#Load module netbackup.pm from current directory
+use lib '.';
+
+use netbackup;
+use strict;
+use warnings;
+use Getopt::Long qw(GetOptions);
+use JSON;
+
+sub printUsage {
+  print "\nUsage : perl get_nb_jobs.pl -nbmaster <master_server> -username <username> -password <password> --filter <filter> --cleanuptime <cleanuptime> [-domainname <domain_name>] [-domaintype <domain_type>] [--verbose]\n\n";
```

Comment:
Use points to get_nb_jobs.

Reply:
Also keep the username password domainname and domaintype together.

Reply:
domainname and domaintype, those are optional fields.

Reply:
Fixed the name of the script.